Library/Formula/python-markdown.rb

```diff
@@ -0,0 +1,18 @@
+class PythonMarkdown < Formula
+  desc "Python implementation of Markdown"
+  homepage "https://pypi.python.org/pypi/Markdown"
+  url "https://pypi.python.org/packages/source/M/Markdown/Markdown-2.6.5.tar.gz"
+  sha256 "8d94cf6273606f76753fcb1324623792b3738c7612c2b180c85cc5e88642e560"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", lib/"python2.7/site-packages"
```

Comment:
I'd prefer for this to be sandboxed in `libexec`, really.

```ruby
ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
```

@tdsmith is free to disagree with me though.